package.json

```
@@ -3,6 +3,10 @@
   "version": "0.2.0",
   "author": "Hiroshiba Kazuyuki",
   "private": true,
+  "engines": {
+    "node": ">=14.16 <15",
+    "npm": ">=6.14.11 <7"
```

今のlockはnpmのバージョン`7.20.2`で作られています。一旦それに合わせたいです。
```suggestion
    "npm": ">=7.20.2 <8"
```

---

npm@7以降はlockfileVersionが2になってしまい、現状と状況が変わらないかと思います

---

node@14のデフォルトはnpm@6で、環境構築の手間が増えるかと思います

---

なるほどです、たしかに簡単に環境が作れたほうが新規開発者に優しそうです。
`npm@6`でいきましょう。